arrow-fx-coroutines/src/test/kotlin/arrow/fx/coroutines/stream/StreamTest.kt

```diff
@@ -768,15 +768,15 @@ class StreamTest : StreamSpec(spec = {
             when (uncons1) {
               null -> Pull.output1(acc)
               else -> Pull.output1(uncons1.head)
-                .flatMap { loop(acc + uncons1.head, uncons1.tail) }
+                .append { loop(acc + uncons1.head, uncons1.tail) }
             }
           }
 
         loop(0, it.asPull()).stream()
       }
 
       Stream.iterate(0, Int::inc)
-        .flatMap { Stream(it).delayBy(20.milliseconds) }
```

Comment:
We've discussed `50.milliseconds` as a minimum delay in another PR. Why the decrease here?